### DISCUSSION TITLE
Submit to package repositories.

### DISCUSSION THREAD

**Discussion by beedell-roke** [oskarsh/Yin-Yang#323] [Ideas]
### Why do you need it?

1.	https://github.com/oskarsh/Yin-Yang/releases doesn't contain installable packages, or binaries.
1.	It's already been requested that it be submitted to a COPR repository at https://github.com/oskarsh/Yin-Yang/discussions/192#discussion-5120085, but that's just short of official maintenance. However, I would consider it adequate.
1.	https://github.com/oskarsh/Yin-Yang/issues/208#issue-1750167294 augments this, but doesn't replace it.

Summarily, https://github.com/oskarsh/Yin-Yang/blob/03bfb2f25a85bef48cc9e1ff1a73cd3c2056376b/README.md#installation is too "scary" (so-to-speak) for novice users.

**Comment by beedell-roke** [comment]
> https://github.com/oskarsh/Yin-Yang/issues/311#event-13766909716

Please remove https://github.com/oskarsh/Yin-Yang/issues?q=label%3A%22plugin+support%22 (and instead add https://github.com/oskarsh/Yin-Yang/issues?q=label%3Aenhancement).

**Comment by l0drex** [comment]
We are currently in the work to release Yin-Yang on flatpak, which will make the installation process a lot easier for a lot of people. However, I do not have the time to create additionally packages for every package system, and each new way of distribution would create one more source of potential bugs, requiring even more time for reproduction, investigation and fixing.
I would appreciate it if someone from the community wants to create and maintain support for a repository. I can offer advice for the process.
